### PR TITLE
Replaced all internal imports of 3rd party pkgs with public API

### PR DESCRIPTION
**Pull request by L2jLiga**
- Replaced internal imports w/ public APIs
- Replaced shell script w/ npm one
- Moved typing for lodash to devDependencies section

### CODE REVIEW

**Comment by HeyRoach** [comment]
run tests

File: angular.json [hunk 1]
```diff
@@ -150,6 +150,5 @@
         }
       }
     }
-  },
-  "defaultProject": "cloudstack-ui"
```
Comment (HeyRoach) [inline angular.json]: leave it here

File: src/app/snapshot/snapshots-page/snapshot-list-item/snapshot-item.component.ts [hunk 2]
```diff
@@ -29,13 +29,12 @@ export class SnapshotItemComponent {
...
     ]
       .filter(state => this.item.state === state)
-      .map(
-        state =>
-          state === SnapshotStates.BackingUp
-            ? 'backing-up'
-            : state === SnapshotStates.BackedUp
-              ? 'backed-up'
-              : state.toLowerCase(),
+      .map(state =>
```
Comment (HeyRoach) [inline src/app/snapshot/snapshots-page/snapshot-list-item/snapshot-item.component.ts]: remove these changes

File: angular.json (hunk 1 above)
Comment (L2jLiga) [reply angular.json]: default project field deprecated and will be removed in next major release

File: src/app/snapshot/snapshots-page/snapshot-list-item/snapshot-item.component.ts (hunk 2 above)
Comment (L2jLiga) [reply src/app/snapshot/snapshots-page/snapshot-list-item/snapshot-item.component.ts]: This changes caused your pre-commit hook

**Comment by dron8552** [comment]
TO TEST:
REQ_VM_LIST_001
REQ_VM_DTLS_014
REQ-LOGS-008

**Pull request merged**